networking/networking-operators-overview.adoc

```diff
@@ -26,4 +26,8 @@ The External DNS Operator deploys and manages ExternalDNS to provide the name re
 
 [id="ingress-node-firewall-operator-1"]
 == Ingress Node Firewall Operator
-The Ingress Node Firewall Operator uses an extended Berkley Packet Filter (eBPF) and eXpress Data Path (XDP) plugin to process node firewall rules, update statistics and generate events for dropped traffic. The operator manages ingress node firewall resources, verifies firewall configuration, does not allow incorrectly configured rules that can prevent cluster access, and loads ingress node firewall XDP programs to the selected interfaces in the rule's object(s). For more information, see xref:../networking/ingress-node-firewall-operator.adoc#ingress-node-firewall-operator[Understanding the Ingress Node Firewall Operator]
\ No newline at end of file
+The Ingress Node Firewall Operator uses an extended Berkley Packet Filter (eBPF) and eXpress Data Path (XDP) plugin to process node firewall rules, update statistics and generate events for dropped traffic. The operator manages ingress node firewall resources, verifies firewall configuration, does not allow incorrectly configured rules that can prevent cluster access, and loads ingress node firewall XDP programs to the selected interfaces in the rule's object(s). For more information, see xref:../networking/ingress-node-firewall-operator.adoc#ingress-node-firewall-operator[Understanding the Ingress Node Firewall Operator]
```

Comment:
This was there. I guess my changes are somehow making Git think that I added this? Not sure why git would think that.

Reply:
this is so weird!

Reply:
I'm pretty sure that's just because you added a new line at the end of this line :-)